ext/project_data.yaml

```diff
@@ -29,7 +29,7 @@ gem_rdoc_options:
 gem_platform_dependencies:
   universal-darwin:
     gem_runtime_dependencies:
-      CFPropertyList: '~> 2.2.6'
+      CFPropertyList: '~> 2.3.5'
```

Comment:
Travis is failing because of the gem conflict.  For now (and until we've cut the next Facter 2.5.0 gem), we're actually going to have to make Puppet less restrictive by changing this to:

```
      CFPropertyList: '~> 2.2'
```

As has been done in https://github.com/puppetlabs/facter/pull/1569

Once Facter 2.5.0 is released to the wild, and before we ship a Puppet gem, we can make Puppet more restrictive once again like we've done here (though it may be beneficial to just let Puppet grab latest in 2 series if available).